server/real_environment/real_environment.go

```diff
@@ -603,3 +603,10 @@ func (r *RealEnv) GetSociArtifactStoreServer() socipb.SociArtifactStoreServer {
 func (r *RealEnv) SetSociArtifactStoreServer(s socipb.SociArtifactStoreServer) {
 	r.sociArtifactStoreServer = s
 }
+
+func (r *RealEnv) GetSociArtifactStoreClient() socipb.SociArtifactStoreClient {
```

Comment:
now that you are initializing the client in podman.go, you no longer need to store it in the env. you can store internally in the podman provider struct

Reply:
sg, done